server/routes/index.js

```diff
@@ -0,0 +1,44 @@
+// Controllers
+const userController = require('../controllers').users;
+
+module.exports = (app, passport) => {
+    // Setup a default catch-all route that sends back a welcome message
+    app.get('/', (req, res) => {
+        return res.status(200).send({
+            message: 'CHEMICAL-X',
+        });
+    });
+
+    app.get('/users', isLoggedIn, userController.list);
+
+    app.post('/users', passport.authenticate('local-signup'),
+        function(req, res) {
+            res.json({message: 'Created new user', user: req.user});
```

Comment:
Maybe a 400 would be better? i.e. if they try signing up with an email that already exists.